flang/test/Driver/lto-flags.f90

```diff
@@ -27,7 +27,7 @@
 ! FULL-LTO: "-fc1"
 ! FULL-LTO-SAME: "-flto=full"
 
-! THIN-LTO-ALL: flang-new: warning: the option '-flto=thin' is a work in progress
+! THIN-LTO-ALL: flang-{{[0-9]+}}: warning: the option '-flto=thin' is a work in progress
```

Comment:
For Clang, lld, and LLVM binary utilities, our convention is to omit the part before `warning:`/`error:`. People may customize the `\d+` part and sometimes symlinks in certain build systems make this more complex.